imports/plugins/core/core/server/no-meteor/schemas/shop.graphql

```diff
@@ -73,6 +76,30 @@ input InviteShopMemberInput {
   shopId: ID!
 }
 
+"Shop URLs to provide for the updateShopUrls mutation"
+input StorefrontUrlsInput {
+  "Storefront Account Profile URL (can include `:accountId` in string)"
+  storefrontAccountProfileUrl: String
+
+  "Storefront Home URL"
+  storefrontHomeUrl: String
+
+  "Storefront single order URL (can include `:orderReferenceId` and `:orderToken` in string)"
+  storefrontOrderUrl: String
+
+  "Storefront orders URL (can include `:accountId` in string)"
+  storefrontOrdersUrl: String
+}
+
+"Input parameters for the updateShopUrls mutation"
+input UpdateShopUrlsInput {
+  "The ID of the shop to which you want to invite this person"
+  shopId: ID!
+
+  "URL object of all URL's for updateShopUrls mutation"
+  storefrontUrls: StorefrontUrlsInput
```

Comment:
I'd add `!` after `StorefrontUrlsInput` because if you don't include it, there's no point in calling the mutation. Also you are missing `clientMutationId` input field.